sklearn/tests/test_common.py

```diff
@@ -82,6 +80,25 @@ def test_all_estimators():
         yield check_parameters_default_constructible, name, Estimator
 
 
+def test_non_meta_estimators():
+    # input validation etc for non-meta estimators
+    # FIXME these should be done also for non-mixin estimators!
```

Comment:
That requires a large amount of work, I think. Or maybe another blacklist? I'm not sure. I prefer small PRs ;)